usvm-ts/src/main/kotlin/org/usvm/TSInterpreter.kt

```diff
@@ -77,7 +77,7 @@ class TSInterpreter(
         val exprResolver = exprResolverWithScope(scope)
 
         val boolExpr = exprResolver
-            .resolveTSExpr(stmt.condition)
+            .resolveTSExprNoUnwrap(stmt.condition)
```

Comment:
Why we don't need to unwrap it?

Reply:
We don't want to lose possible UJoinedBoolExpr for futher fork here